### PR TITLE
Bump ledgerhq deps

### PR DESCRIPTION
**Pull request by chabroA**
## 🦒 Context (issues, jira)
Update following deps:

- `@ledgerhq/hw-transport-node-hid-singleton`: `"6.20.0"` -> `"6.23.0"`
- `@ledgerhq/live-common`: `"^21.26.1"` -> `"^21.27.0"`

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [ ] a specific test planned is defined on Jira
  - [x] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: 
  - [x] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [x] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

### CODE REVIEW

**Comment by juan-cortes** [comment]
This includes the work from https://github.com/LedgerHQ/ledger-live-desktop/pull/4647 which is currently in QA by @alalmi-ledger so merging this should be done only after it's qa ok over there. 

Closes https://github.com/LedgerHQ/ledger-live-desktop/pull/4647

**Comment by juan-cortes** [comment]
QA seem to have found some issues with a specific device, we need more investigation on the cause

**Comment by juan-cortes** [comment]
Reverting the changes on the singleton transport until we figure out what's causing the erratic connection QA found. Can't afford introducing a bug like this that would impact many users. So, we need https://github.com/LedgerHQ/ledgerjs/pull/770 merged, ledgerjs released with that, and to use the new version here again.

Or in the meantime, drop the singleton library bump if you need the rest.

File: package.json [hunk 1]
```diff
@@ -47,9 +47,9 @@
     "@ledgerhq/errors": "6.10.0",
     "@ledgerhq/hw-transport": "6.20.0",
     "@ledgerhq/hw-transport-http": "6.20.0",
-    "@ledgerhq/hw-transport-node-hid-singleton": "6.20.0",
+    "@ledgerhq/hw-transport-node-hid-singleton": "6.23.0",
     "@ledgerhq/ledger-core": "6.14.5",
-    "@ledgerhq/live-common": "^21.26.1",
+    "@ledgerhq/live-common": "^21.27.0",
```
Comment (juan-cortes) [inline package.json:52]: QA KO makes this not possible

**Review by juan-cortes — CHANGES_REQUESTED** [review]

**Comment by chabroA** [comment]
> Reverting the changes on the singleton transport until we figure out what's causing the erratic connection QA found. Can't afford introducing a bug like this that would impact many users. So, we need [LedgerHQ/ledgerjs#770](https://github.com/LedgerHQ/ledgerjs/pull/770) merged, ledgerjs released with that, and to use the new version here again.
> 
> Or in the meantime, drop the singleton library bump if you need the rest.

@juan-cortes Ok so if I understand correctly we can bump `@ledgerhq/live-common: "^21.26.1" -> "^21.27.0"` but not `@ledgerhq/hw-transport-node-hid-singleton`?
If that the case, that's fine by me 👌

**Comment by chabroA** [comment]
Duplicated by https://github.com/LedgerHQ/ledger-live-desktop/pull/4664

**Pull request closed without merging**